src/components/Modal.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
+import './Modal.css';
+
+function Modal() {
```

Comment:
aici ai putea folosi arrow function, ii total correct si asa, just i preffer to use this way: `const Modal = () => {}` ✌️